plugins/goodreads.koplugin/main.lua

```diff
@@ -0,0 +1,226 @@
+local InputContainer = require("ui/widget/container/inputcontainer")
+local InputDialog = require("ui/widget/inputdialog")
+local DoubleKeyValuePage = require("doublekeyvaluepage")
+local MultiInputDialog = require("ui/widget/multiinputdialog")
+local InfoMessage = require("ui/widget/infomessage")
+local UIManager = require("ui/uimanager")
+local Screen = require("device").screen
+local DEBUG = require("dbg")
+local _ = require("gettext")
+
+local GoodReads = InputContainer:new {
+    goodreadersKey = "",
```

Comment:
We should keep the coding style consistent with other places, i.e. `camelCase` for function name and `under_score` for variable name.